installing/installing_azure/installing-azure-customizations.adoc

```diff
@@ -61,6 +57,10 @@ include::modules/installation-configure-proxy.adoc[leveloffset=+2]
 
 * For more details about Accelerated Networking, see xref:../../machine_management/creating_machinesets/creating-machineset-azure.adoc#machineset-azure-accelerated-networking_creating-machineset-azure[Accelerated Networking for Microsoft Azure VMs].
 
+include::modules/installation-user-defined-tags-azure.adoc[leveloffset=+1]
```

Comment:
Maybe consider removing "the" from the heading here, to make it align with the next heading? Your call, though.

Reply:
Good suggestion. Though this isn't in scope of this change, I will fix it.

Reply:
is it ok to be marked as peer-review-done?